charts/pipelines-library/templates/tasks/sonar/sonarqube-dotnet.yaml

```diff
@@ -82,14 +82,23 @@ spec:
       script: |
         set -e
 
-        PROJECT_EXISTS=$(curl -s -u ${SONAR_TOKEN}: "${SONAR_HOST_URL}/api/components/show?component=${SONAR_PROJECT_KEY}" | jq -r ".component.key")
+        SONAR_RESPONSE=$(curl -s -u "${SONAR_TOKEN}:" "${SONAR_HOST_URL}/api/components/show?component=${SONAR_PROJECT_KEY}")
```

Comment:
can we just keep the previous logic and rely on 

set -eo pipefail

and fail flag for curl?